testnet/03e6b8e88efc0f3194980e8810263da9ee243d28ac6cf367f618fd24b19a5b7aac.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "Ghost",
```

The `id` field is missing.

```suggestion
  "id": 161,
  "name": "Ghost",
```